tests/integration/helpers/element-test.js

```diff
@@ -89,7 +89,7 @@ module('Integration | Helper | element', function(hooks) {
         {{#let (element "button") as |Tag|}}\
           <Tag type="button" id="action" {{on "click" this.didClick}}>hello world!</Tag>\
         {{/let}}\
-      ', { insertRuntimeErrors: true }));
```

Comment:
I removed this as Embroider seems to have its own `hbs` babel-plugin, that [throws](https://github.com/embroider-build/embroider/blob/master/packages/core/src/babel-plugin-inline-hbs.ts#L123) when receiving more than one argument. I am not sure what this option does here, so please advice if/how this needs to be fixed!?